.ci/build_demo.sh

```diff
@@ -127,7 +127,7 @@ echo -e "${BLUE}Building the application...${NC}"
 sleep 10
 
 # Build the application inside php-fpm container
-docker-compose exec -T php-fpm ./phing db-create db-demo grunt error-pages-generate microservice-product-search-recreate-structure microservice-product-search-export-products warmup
```

Comment:
hmmmmmm. I am not sure but what about this way?
```suggestion
docker-compose exec -T php-fpm ./phing db-create db-demo grunt error-pages-generate product-search-recreate-structure product-search-export-products warmup
```

Reply:
what about `docker/conf/docker-compose.demo.yml.dist` ????? MICROSERVICES, ELASTIC_SEARCH_INDEX_PREFIX and stuffs ?

Reply:
https://github.com/shopsys/demoshop/blob/mc-beta6-upgrade/docker/conf/docker-compose.demo.yml.dist

Reply:
you also need to update `.ci/build_demo.sh`

Reply:
hmmm @MattCzerner didnt you forget some review messages?